ivy/functional/frontends/paddle/tensor/math.py

```diff
@@ -192,3 +192,9 @@ def exp(x, name=None):
 @to_ivy_arrays_and_back
 def reciprocal(x, name=None):
     return ivy.reciprocal(x)
+
+
+@with_unsupported_dtypes({"2.4.2 and below":("float16", "bfloat16")}, "paddle")
+@to_ivy_arrays_and_back
+def floor_divide(x1,x2,/,*,out = None, name = None):
+    return ivy.floor_divide(x1,x2, out = out)
```

Comment:
@pranjalkar99 Changes look good! Just a few minor changes required: for the parameters of floor_divide and ivy.floor_divide, there should be a space after commas (just like in regular grammar). Also, the equal signs should be right next to the parameters and values (e.g. out=None) instead of having a space in between.